src/Oro/Bundle/UserBundle/Controller/UserController.php

```diff
@@ -50,9 +50,17 @@ public function updateProfileAction()
 
     /**
      * @AclAncestor("pim_user_user_edit")
+     *
+     * @param int $id
+     *
+     * @return JsonResponse|\Symfony\Component\HttpFoundation\Response
      */
-    public function apigenAction(User $user)
+    public function apigenAction($id)
     {
+        $userRepository = $this->container->get('pim_user.repository.user');
```

Comment:
It is a Oro controller, extending `Symfony\Bundle\FrameworkBundle\Controller\Controller`